unstructured_api_tools/pipelines/convert.py

```diff
@@ -56,6 +58,11 @@ def _infer_params_from_pipeline_api(script: str) -> Dict[str, Optional[Any]]:
     """A helper function to prepare jinja interpolation.
     Returns a list of string (multi-value) parameters to expose in the FastAPI route.
     """
+    if script.count("def pipeline_api(") > 1:
```

Comment:
could you add tests for these two cases? you can test the logger info with `caplog` from `pytest`.

Reply:
Sure, I have added. Thanks for the suggestion!